jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/ServletContextHandlerTest.java

```diff
@@ -1900,15 +1901,26 @@ public void contextDestroyed(ServletContextEvent sce)
                     contextDestroy.set(true);
             }
         });
-        ServletHandler shandler = servletContextHandler.getServletHandler();
-
         ResourceHandler rh = new ResourceHandler();
         rh.setBaseResource(ResourceFactory.of(rh).newResource(Paths.get(".")));
-
         servletContextHandler.insertHandler(rh);
-        assertEquals(shandler, servletContextHandler.getServletHandler());
-        assertEquals(rh, servletContextHandler.getHandler());
-        assertEquals(rh.getHandler(), shandler);
+
+        Handler last = new Handler.Abstract()
+        {
+            @Override
+            public boolean handle(Request request, Response response, Callback callback) throws Exception
+            {
+                return false;
+            }
+        };
+        servletContextHandler.getTail().setHandler(last);
```

Comment:
See above, this line would more naturally be `servletContextHandler.setHandler(last)`.

Reply:
@sbordet  ditto - that has the effect of replacing the sessionHandler/securityHandler/servletHandler that this class has provided. inserting handlers makes sense, but replacing breaks the class.   This is a warning we've had in jetty-9/10/11 for sometime.

I could replace it with:
```suggestion
        servletContextHandler.getServletHandler().setHandler(last);
```

But that assumes there is not already a handler set on the servlet handler.  This is more robust way of doing it